dChatFilter/dChatFilter.cpp

```diff
@@ -134,8 +153,4 @@ size_t dChatFilter::CalculateHash(const std::string& word) {
 	size_t value = hash(word);
 
 	return value;
-}
-
-bool dChatFilter::IsInWordlist(size_t word) {
-	return std::find(m_Words.begin(), m_Words.end(), word) != m_Words.end();
-}
+}
```

Comment:
Having new lines at end of files is ideal

Reply:
Why?

Reply:
Because it is standard for how some files read to EOF
c.f. https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#:~:text=If%20content%20is%20added%20to,you%20actually%20wanted%20to%20see.

Reply:
Done